### PR TITLE
Apim 3.18 audit braking changes

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
https://github.com/gravitee-io/issues/issues/7536

**Description**
Add braking changes for audit management API

**Screenshots**

**Additional context**

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/apim-3-18-audit-braking-changes/index.html)

### CODE REVIEW

File: pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc [hunk 1]
```diff
@@ -1,5 +1,19 @@
 = Upgrade to 3.18.0
 
+
+== Breaking changes
```
Comment (ThibaudAV) [inline pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc:4]: 🤔  to add or automatically added by the real release ?

File: pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+
+=== Environment Audits
+
+Before this version APIM management API will allow to get all audits of all environments and organization with this endpoint : `[GET] management/organizations/{orgId}/environments/{envId}/apis/{api}/audit`.  
+
+Now this GET always return audit of the environment added in the url and no longer returns organization audits. 
+To get organization audits you need to use the following endpoint : `[GET] management/organizations/{orgId}/audit`.
+
+The query params `envLog:boolean` & `orgLog:boolean` are removed. 
+Now to get only environment or organization audit you have to use the query params `referenceType:string` with `ENVIRONMENT` or `ORGANIZATION`. You can also filter by environment id with `environment:string` query params.
```
[the remark this answers is not in the document]
Comment (ThibaudAV) [reply pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc:14]: o nice. added

File: pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc [hunk 3]
```diff
@@ -1,5 +1,21 @@
 = Upgrade to 3.18.0
 
+
+== Breaking changes
+
+=== Environment Audits
+
+Endpoint :`[GET] management/organizations/{orgId}/environments/{envId}/apis/{api}/audit` (link:{{ '/apim/3.x/management-api/' | relative_url | append: current_version | append: '/index.html#operation/getApiAudits' }}[API audits^]).
+
+Before this version APIM management API will allow to get all audits of all environments and organization with this endpoint.
```
Comment (gaetanmaisse) [inline pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc:10]:
```suggestion
Before this version, the Management API allowed getting all audits of all environments and organizations with this endpoint.
```

File: pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc [hunk 4]
```diff
@@ -1,5 +1,21 @@
 = Upgrade to 3.18.0
 
+
+== Breaking changes
+
+=== Environment Audits
+
+Endpoint :`[GET] management/organizations/{orgId}/environments/{envId}/apis/{api}/audit` (link:{{ '/apim/3.x/management-api/' | relative_url | append: current_version | append: '/index.html#operation/getApiAudits' }}[API audits^]).
```
Comment (gaetanmaisse) [inline pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc:8]:
```suggestion
Endpoint:`[GET] management/organizations/{orgId}/environments/{envId}/apis/{api}/audit` (link:{{ '/apim/3.x/management-api/' | relative_url | append: current_version | append: '/index.html#operation/getApiAudits' }}[API audits^]).
```

File: pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc [hunk 5]
```diff
@@ -1,5 +1,21 @@
...
+
+Before this version APIM management API will allow to get all audits of all environments and organization with this endpoint.
+
+Now this GET always return audit of the environment added in the url and no longer returns organization audits. 
+To get organization audits you need to use the following endpoint : `[GET] management/organizations/{orgId}/audit`.
```
Comment (gaetanmaisse) [inline pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc:13]:
```suggestion
To get organization audits you need to use the following endpoint: `[GET] management/organizations/{orgId}/audit`.
```

File: pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc [hunk 6]
```diff
@@ -1,5 +1,21 @@
...
+
+== Breaking changes
+
+=== Environment Audits
+
+Endpoint :`[GET] management/organizations/{orgId}/environments/{envId}/apis/{api}/audit` (link:{{ '/apim/3.x/management-api/' | relative_url | append: current_version | append: '/index.html#operation/getApiAudits' }}[API audits^]).
+
+Before this version APIM management API will allow to get all audits of all environments and organization with this endpoint.
+
+Now this GET always return audit of the environment added in the url and no longer returns organization audits.
```
Comment (gaetanmaisse) [inline pages/apim/3.x/installation-guide/upgrades/3.18.0/README.adoc:12]:
```suggestion
As of 3.18.0, a GET request will return the audits of the environment specified in the url and no longer return organization audits. 
```

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**